source/components/molecules/EditableList/EditableList.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+import { Text, Button } from '../../atoms';
+
+/**
+ * EditableList
+ * A Molecule Component to use for rendering a list with the possibility of editing the list values.
+ */
+
+const EditableListWrapper = styled.View`
+  background: red;
+  display: flex;
+  flex-direction: column;
+
+  width: 382px;
+  height: auto;
+
+  background: #fbf7f0;
+  border-radius: 9.5px;
+  overflow: hidden;
+`;
+
+const EditableListHeader = styled.View`
+  background: #f5e0d8;
```

Comment:
All the colors have fixed values. Try to make them dynamic , so that we can use this component in other pages. :)

Reply:
Good catch! :D